tests/prmbackslash_2.prm

```diff
@@ -33,7 +33,7 @@ set Surface pressure\
 set Use years in out\
 put instead of seconds = \
 false  # default: true
-set Nonlinear solver\
+set Nonlinear solver \
```

Comment:
so are you saying that we previously accepted no space plus backslash, but that we will require the backslash after the test?